bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/TypeConverter.java

```diff
@@ -935,7 +938,7 @@ public static Boolean stringToBoolean(String value) throws NumberFormatException
         throw new NumberFormatException();
     }
 
-    public static BDecimal stringToDecimal(String value) throws NumberFormatException {
+    public static DecimalValue stringToDecimal(String value) throws NumberFormatException {
```

Comment:
As these methods are used in our APIs, shall we keep it as `BDecimal`? The other methods seems to be returning the instances of `io.ballerina.runtime.api` package. Eg: `stringToChar()`, `stringToXml` etc.
Is there a reason for this change?

Reply:
`BDecimal` needs to be restored as return type for `getDecimalValue` method in `CliUtils` as well.